### PR TITLE
[Jsontracer] Add ruler to render timeline of bar-graph

### PR DESCRIPTION
**Pull request by wnsdud4197**
This will render ruler to show timeline of bar-graph

ONE-vscode-DCO-1.0-Signed-off-by: Junyeong Yang <wnsdud4197@naver.com>

### CODE REVIEW

**Review by parkjisu6239 — DISMISSED** [review]

**Comment by wnsdud4197** [comment]
from draft #126
from issue #94

**Review by parkjisu6239 — APPROVED** [review]

File: media/Jsontracer/ruler.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      }
+
+      graduation.append(smallGraduation);
+    }
+
+    ruler.append(graduation);
+  }
+}
+
+function calculateGraduation(graduation) {
```
Comment (llFreetimell) [inline media/Jsontracer/ruler.js:91]: I saw exactly same code in `media/Jsontracer/dynamicGraduation.js`.
Could you add `TODO Reduce duplicate function` ?

Comment (wnsdud4197) [reply media/Jsontracer/ruler.js:91]: OK, i will add `TODO` and apply that to the next PR : )

**Review by parkjisu6239 — APPROVED** [review]

File: media/Jsontracer/ruler.js (hunk 1 above)
Comment (llFreetimell) [reply media/Jsontracer/ruler.js:92]: Since this is not that urgent, you can do it later :D

**Review by llFreetimell — APPROVED** [review]
LGTM

**Pull request merged**